crates/uv/tests/common/mod.rs

```diff
@@ -24,6 +24,9 @@ use uv_interpreter::{find_requested_python, PythonVersion};
 // Exclude any packages uploaded after this date.
 pub static EXCLUDE_NEWER: &str = "2024-03-25T00:00:00Z";
 
+pub const PACKSE_FIND_LINKS_URL: &str =
```

Comment:
Should we call it like `BUILD_VENDOR_LINKS_URL` to make it clearer whats in here?

Reply:
It'd be nice to document why it's useful too. I added this so we could use `--index-url` instead of `--extra-index-url` to prevent dependency confusion attacks against our test suite, but without vendoring build dependencies that will generally fail.